rs/nns/governance/src/governance.rs

```diff
@@ -2313,7 +2313,7 @@ impl Governance {
                             && neuron.visibility() == Some(Visibility::Public)
                         );
                 if let_caller_read_full_neuron {
-                    let mut proto = NeuronProto::from(neuron.clone());
+                    let mut proto = neuron.clone().into_proto(now);
```

Comment:
Because conversion now requires some additional auxiliary data, there are a whole bunch of diffs like this.